js/src/figure.js

```diff
@@ -832,38 +766,12 @@ var FigureView = widgets.DOMWidgetView.extend( {
         this.model.on('change:height', this.update_size, this);
         this.model.on('change:displayscale', this.update_size, this);
 
-        this.model.on('change:ambient_coefficient', this.update_light, this);
-        this.model.on('change:diffuse_coefficient', this.update_light, this);
-        this.model.on('change:specular_coefficient', this.update_light, this);
-        this.model.on('change:specular_exponent', this.update_light, this);
-        var update_volume_minmax = () => {
-            this.box_material_volr.uniforms.volume_data_range.value = [this.model.get('volume_data_min'), this.model.get('volume_data_max')]
-            this.box_material_volr.uniforms.volume_show_range.value = [this.model.get('volume_show_min'), this.model.get('volume_show_max')]
-            this.update()
-        }
-        this.model.on('change:volume_data_min change:volume_data_max change:volume_show_min change:volume_show_max', update_volume_minmax, this);
-        update_volume_minmax()
-
-        var update_volume_clamp = () => {
-            this.box_material_volr.uniforms.clamp_min.value = this.model.get('volume_clamp_min')
-            this.box_material_volr.uniforms.clamp_max.value = this.model.get('volume_clamp_max')
-            this.update()
-        }
-        this.model.on('change:volume_clamp_min change:volume_clamp_max', update_volume_clamp, this);
-        update_volume_clamp()
-
```

Comment:
What do you mean by a maximum? you will not go over the maximum? isn't it all defined in your transfer function?

But I removed this just to make my current way of volume rendering to work. And yes if we further look into it can be integrate again if its needed for the current ipyvolume volume rendering.

Reply:
Lets say your data is in the range [0, 10], however you may want to focus only on the [0, 7] range. For instance, there may be an outlier at 10. Does that make sense?